README.md

```diff
@@ -62,9 +63,9 @@ You can pass options to the [cookie parse](https://github.com/jshttp/cookie#cook
 
 ### Sending
 
-The method `setCookie(name, value, options)` is added to the `reply` object
+The method `setCookie(name, value, options)` and its alias `cookie(name, value, options)` are added to the `reply` object
 via the Fastify `decorateReply` API. Thus, in a request handler,
-`reply.setCookie('foo', 'foo', {path: '/'})` will set a cookie named `foo`
+`reply.setCookie('foo', 'foo', {path: '/'})` or `reply.cookie('foo', 'foo', {path: '/'})` will set a cookie named `foo`
```

Comment:
```suggestion
`reply.setCookie('foo', 'foo', {path: '/'})` will set a cookie named `foo`
```

I think mentioning the alias is available is enough. We don't need to write the example twice because that is hard to read.